lib/src/main/java/com/thenewboston/common/http/bankapi/bankdto/BankDTO.kt

```diff
@@ -0,0 +1,19 @@
+package com.thenewboston.common.http.bankapi.bankdto
+
+import com.google.gson.annotations.SerializedName
+
+data class BankDTO(
+    @SerializedName("account_number")
+    val accountNumber: String,
+    @SerializedName("ip_address")
+    val ipAddress: String,
+    @SerializedName("node_identifier")
+    val nodeIdentifier: String,
+    val port: Int? = null,
```

Comment:
is nullability on the BankDTO intended?

Reply:
[https://www.thenewboston.com/bank-api/banks](url)
In the example response port has the value null